crates/api_models/src/payments.rs

```diff
@@ -1120,31 +1120,64 @@ pub enum SessionToken {
     },
     /// The session response structure for Apple Pay
     Applepay {
-        /// Timestamp at which session is requested
-        epoch_timestamp: u64,
-        /// Timestamp at which session expires
-        expires_at: u64,
-        /// The identifier for the merchant session
-        merchant_session_identifier: String,
-        /// Applepay generates unique ID (UUID) value
-        nonce: String,
-        /// The identifier for the merchant
-        merchant_identifier: String,
-        /// The domain name of the merchant which is registered in Apple Pay
-        domain_name: String,
-        /// The name to be displayed on Apple Pay button
-        display_name: String,
-        /// A string which represents the properties of a payment
-        signature: String,
-        /// The identifier for the operational analytics
-        operational_analytics_identifier: String,
-        /// The number of retries to get the session response
-        retries: u8,
-        /// The identifier for the connector transaction
-        psp_id: String,
+        /// Session object for Apple Pay
+        session_object: Box<ApplePaySessionObject>,
```

Comment:
Boxed all the field in each variant of the enum, or create object and for each variant and box them.

Reply:
Done